sites/partners/src/pages/index.tsx

```diff
@@ -57,6 +75,20 @@ class ListingsLink extends formatLinkCell {
   }
 }
 
+export const getFlagInAllJurisdictions = (
```

Comment:
question: are you not able to use the `doJurisdictionsHaveFeatureFlagOn` function from the authContext? That has the ability to check if it's set to true on all jurisdictions the user has access to or just one.

Reply:
@emilyjablonski have you had a chance to look at this?

Reply:
Updated! Lmk whatcha think

Reply:
Thanks for making this change!